analysis/monkmistweaver/src/modules/spells/EssenceFont.tsx

```diff
@@ -109,6 +144,23 @@ class EssenceFont extends Analyzer {
     this.targetOverlap += 1;
   }
 
+  handleEndChannel(event: EndChannelEvent) {
+    if (
+      event.ability.guid === SPELLS.ESSENCE_FONT.id &&
+      event.duration < this.expected_duration - this.cancelDelta
+    ) {
+      console.log(
```

Comment:
Console should only print if it's a major system breaking issue
You can add a debug boolean set to false if you want the next developer to have a debug toggle if you want BUT console is holy when it goes to prod

Reply:
Removed